pkgs/development/tools/comby/default.nix

```
@@ -0,0 +1,50 @@
+{ pkgs, ocamlPackages, fetchFromGitHub, lib }:
```

Take advantage of `callPackage`.

```suggestion
{  ocamlPackages, fetchFromGitHub, lib, zlib, pkg-config }:
```